packages/base-map/__mocks__/SelectVehicles.js

```diff
@@ -20,14 +22,39 @@ class SelectVehicles extends MapLayer {
     vehicles: vehicleData
   };
 
+  constructor(props) {
+    super(props);
+
+    this.onOverlayAdded = this.onOverlayAdded.bind(this);
+    this.onOverlayRemoved = this.onOverlayRemoved.bind(this);
+    this.onViewportChanged = this.onViewportChanged.bind(this);
```

Comment:
These bindings can be done by rewriting the class methods as follows:

```js
onOverlayAdded = e => {
  ...
}
```

Reply:
I made the change and added a comment in that example layer stating that both syntaxes are supported.